### ISSUE TITLE
Can't group anymore?

### ISSUE THREAD

**Issue by AmitMY** [ceolter/ag-grid-enterprise#43]
For some reason, it seems that in the new versions (5+) I cannot group columns anymore.
Is there a reason for it, or is it a bug? (Every time I drag them to the area, I see a no entrance icon)

**Comment by randdusing** [comment]
I think you need to set this property to true on the column definition: ```enableRowGroup```

**Comment by AmitMY** [comment]
Shouldn't 'enableRowGroup' default to true, for backwards compatibility? (and for comfort? I want all of my columns in all of my tables groupable, and not one by one)

**Comment by ceolter** [comment]
correct, now you need enableRowGroup, this was mentioned in the change log. and yes it's a breaking change. i played with having it the other way around, however it makes sense to (for example) mark all your value columns as enableValue, and all of your dimension columns as enableRowGroup and / or enablePivot. that would just be backwards. so yes it's a breaking change, but for good reason. going forward, you will want to create a report and say "these 5 columns are dimensions, so i want to allow pivot and group, and these 10 columns are numbers, so i want to allow aggregations, and this column is a timestamp, so i will leave that alone as it should be aggregated on or over"

**Comment by AmitMY** [comment]
Ok, I understand. Any way to have a grid property like 'defaultGroupValue: true' so the default will allow grouping all columns?
I do understand the rational, but for example in my case, I have a report maker, with ability to create dynamic columns out of 100 data columns. In this case, it is weird to have all grouping off, because it is possible to create any combination of any N columns even using if, or javascript code in the valueGetter..

TL;DR:
Any chance adding ability to change defaultGroupValue?

Thanks!

**Comment by ceolter** [comment]
```Any way to have a grid property like 'defaultGroupValue: true' so the default will allow grouping all columns``` really what is needed is a way to provide a default colDef, that can have absolutely anything any colDef can have, that is just copied as a starting point for each column

```Any chance adding ability to change defaultGroupValue?```
=> will add to my list

**Comment by AmitMY** [comment]
You think big! A default colDef would be huge for the grid, I would love to have it
Hope this can be done in the upcoming weeks
